### PR TITLE
AC-419: Fill up Troubleshooting

### PR DESCRIPTION
**Pull request by adtserapio**
Hi! I have already updated based on comments @f4ww4z. I am very sorry however, as I edited the changes on the browser, it seemed to not have synced properly.

## Description of what I changed

I have added a guide (troubleshooting.md) file that documents login issues and how to file issues This is a continuation of the PR filed at #28. I have made a new PR since the branch I was working on had rebase issues. However, I have already revised based on the comments by @f4ww4z. :)

## Issue I worked on

See https://issues.openmrs.org/browse/AC-419
See https://codein.withgoogle.com/dashboard/task-instances/6683284093272064/

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+However, if there are other issues you come across, you could take the following steps:
+
+#### 1. Visit OpenMRS Talk 
+
+You could check out current issues by searching "android client" as seen in the picture below. 
+![OpenMRS Talk Android Client Issues](assets/openmrs_talk_android_client.png)
+
+If not, you could file an issue as seen below. 
+
+![OpenMRS Talk Android Client Filing Issue](assets/openmrs_talk_android_client_file_issue.png)
```
Comment (f4ww4z) [inline troubleshooting.md:27]: Can you hide the other irrelevant tabs in the screenshot? Sorry for not noticing this earlier.

File: troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Troubleshooting Android Client
+
+#### Can't Login?
+
+There are a few things you would need to first take note in logging in. Make sure that you enter a valid server for the URL address. In setting up a customized server, you may request one from helpdesk or server maintainers. Due to a limitation of resources, do note however that these customized test servers are only meant to be used for a limited time only. For more information regarding different testing servers, please visit the following [link](https://wiki.openmrs.org/display/ISM/OpenMRS+environments). 
+
+You could also refer to the part of the guide regarding features, which discusses how to test on the default test server [link](hhttps://openmrs.github.io/openmrs-android-client-user-guide/features.html).
```
Comment (f4ww4z) [inline troubleshooting.md]:
```suggestion
You may also refer to [this guide](https://openmrs.github.io/openmrs-android-client-user-guide/features.html) which discusses how to test on the default test server.
```

Comment (f4ww4z) [reply troubleshooting.md]: You miss typed https :)

File: troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+![OpenMRS Talk Android Client Filing Issue](assets/openmrs_talk_android_client_file_issue.png)
+
+#### 2. File in JIRA 
+
+As an OpenMRS developer, you could opt to file an issue in the JIRA as seen below. If you are new to the OpenMRS environment, you would need to go to helpdesk and ask for JIRA access first.
+
+![OpenMRS JIRA Android Client Filing Issue](assets/openmrs_jira_android_client_file_issue.png)
+
+
```
Comment (f4ww4z) [inline troubleshooting.md]: Newline is not needed here.

**Review by f4ww4z — CHANGES_REQUESTED** [review]
Thanks for the changes @Pilipino . I have some additional comments, then you should be good to go.

**Comment by adtserapio** [comment]
I have updated @f4ww4z! Please advise! :D

**Review by f4ww4z — APPROVED** [review]

**Pull request merged**